pkgs/development/libraries/newt/default.nix

```
@@ -9,11 +11,18 @@ stdenv.mkDerivation rec {
     sha256 = "0cdvbancr7y4nrj8257y5n45hmhizr8isynagy4fpsnpammv8pi6";
   };
 
+  postUnpack = lib.optionalString pythonSupport ''
+    substituteInPlace $sourceRoot/configure \
+      --replace "/usr/include/python" "${python}/include/python"
```

In case python ever splits its outputs
```suggestion
      --replace "/usr/include/python" "${lib.getDev python}/include/python"
```